containers/Loan/Borrow/index.tsx

```diff
@@ -56,7 +56,7 @@ class LoanBorrow extends React.Component<Props, State> {
     const { loan, analytics } = this.props;
 
     const ceilingSet = loan.principal.toString() !== '0';
-    const availableFunds = analytics && analytics.data && analytics.data.availableFunds || '0';
+    const availableFunds = analytics && analytics.data && analytics.availableFunds || '0';
```

Comment:
same as for the minJuniorRatio availableFunds have to be part of data

Reply:
Changed